vita3k/gui/src/profiles_manager_dialog.cpp

```diff
@@ -18,49 +18,60 @@
 #include "private.h"
 
 #include <config/functions.h>
-#include <host/state.h>
 #include <misc/cpp/imgui_stdlib.h>
 #include <util/log.h>
 
 namespace gui {
+    
+void init_user(GuiState &gui, HostState &host) {
+    if (host.cfg.users.find(host.cfg.current_user_id) == host.cfg.users.end())
+        host.cfg.current_user_id = host.cfg.users.begin()->first;
+
+    if (host.io.user_id.empty() || std::stoi(host.io.user_id) != host.cfg.current_user_id)
+        host.io.user_id = fmt::format("{:0>2d}", host.cfg.current_user_id);
+}
+
+static std::string online_id;
```

Comment:
:(

Reply:
Where is problèm ? 🤔

Reply:
I want these things in GuiState but I don't think it will happen so instead i will just leave a disapproving face each time it happens.